apis/youtube/handler.go

```diff
@@ -0,0 +1,38 @@
+package youtube
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Handler allows to handle Youtube-related operations properly
+type Handler struct {
+	api *API
+}
+
+// NewHandler returns a new Handler instance
+func NewHandler(api *API) *Handler {
+	return &Handler{
+		api: api,
+	}
+}
+
+// GetChannel returns the description of the user having the given user id
+func (h *Handler) GetChannel(userID string) (Channel, error) {
+	// Check the validity of the id
+	if strings.ContainsRune(userID, ',') {
+		return Channel{}, fmt.Errorf("invalid user id: %s", userID)
+	}
+	id := getChannelID(userID)
+	channel, err := h.api.GetChannel(id)
```

Comment:
It's always better to not declare variables if they're used only once and they don't provide any improvement to the code readability
```suggestion

	channel, err := h.api.GetChannel(getChannelID(userID))
```